Source/GTMSessionFetcher.h

```diff
@@ -624,7 +580,6 @@ NSData *_Nullable GTMDataFromInputStream(NSInputStream *inputStream, NSError **o
 - (nullable id<NSURLSessionDelegate>)sessionDelegate;
 - (nullable NSDate *)stoppedAllFetchersDate;
 
-// Methods for compatibility with the old GTMHTTPFetcher.
 @property(atomic, readonly, strong, nullable) NSOperationQueue *delegateQueue;
```

Comment:
This will take some checking for a future cl, but we likely should consider pruning these.

Reply:
Yeah, I wasn't confident enough in removing that one for this PR. Will take a deeper look later; figured it was harmless enough for now.